{
  "uid" : "2572e3883aa7d889",
  "name" : "test_add_exc[d1]",
  "fullName" : "testcase.test_add#test_add_exc",
  "historyId" : "6c90bdb9a20bffeee4e26e7f2f5dc4d0",
  "time" : {
    "start" : 1685791298860,
    "stop" : 1685791298860,
    "duration" : 0
  },
  "status" : "skipped",
  "statusMessage" : "Skipped: unconditional skip",
  "statusTrace" : "('D:\\\\python_Code\\\\pythonProject\\\\pythonclasses\\\\autoclass\\\\day06\\\\appautotest\\\\testcase\\\\test_add.py', 18, 'Skipped: unconditional skip')",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 78,
  "retriesStatusChange" : true,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1685791298863,
      "stop" : 1685791298974,
      "duration" : 111
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "base_url",
    "time" : {
      "start" : 1685791298974,
      "stop" : 1685791298975,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "setup_teardown",
    "time" : {
      "start" : 1685791298975,
      "stop" : 1685791313253,
      "duration" : 14278
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "_verify_url",
    "time" : {
      "start" : 1685791298975,
      "stop" : 1685791298975,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "sensitive_url",
    "time" : {
      "start" : 1685791298975,
      "stop" : 1685791298975,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "_skip_sensitive",
    "time" : {
      "start" : 1685791298975,
      "stop" : 1685791298975,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "afterStages" : [ {
    "name" : "setup_teardown::0",
    "time" : {
      "start" : 1685791324675,
      "stop" : 1685791325625,
      "duration" : 950
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.skip"
  }, {
    "name" : "parentSuite",
    "value" : "testcase"
  }, {
    "name" : "suite",
    "value" : "test_add"
  }, {
    "name" : "host",
    "value" : "syzpc"
  }, {
    "name" : "thread",
    "value" : "19628-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "testcase.test_add"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "d",
    "value" : "{'name': 'case002_参数化加法', 'num1': 70, 'num2': 32}"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "eaa2b37be1776846",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685790504451,
        "stop" : 1685790504451,
        "duration" : 0
      }
    }, {
      "uid" : "7454a017df0c1f4f",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685789850735,
        "stop" : 1685789850735,
        "duration" : 0
      }
    }, {
      "uid" : "6a7bceab8423aab6",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685789701560,
        "stop" : 1685789701560,
        "duration" : 0
      }
    }, {
      "uid" : "2023404b94fcd0ff",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685789385131,
        "stop" : 1685789385131,
        "duration" : 0
      }
    }, {
      "uid" : "16eff7e3984f65d2",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685789304589,
        "stop" : 1685789304589,
        "duration" : 0
      }
    }, {
      "uid" : "c8b61df06506f79c",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685789033317,
        "stop" : 1685789033317,
        "duration" : 0
      }
    }, {
      "uid" : "6672080618bc5580",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685788653410,
        "stop" : 1685788653410,
        "duration" : 0
      }
    }, {
      "uid" : "559f7cbb4c43c0b0",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685788568686,
        "stop" : 1685788568686,
        "duration" : 0
      }
    }, {
      "uid" : "b31923d46539909f",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685788395299,
        "stop" : 1685788395299,
        "duration" : 0
      }
    }, {
      "uid" : "f4ec0ad96fd2c54f",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685788326498,
        "stop" : 1685788326498,
        "duration" : 0
      }
    }, {
      "uid" : "dd36363a5ebeb675",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685787986456,
        "stop" : 1685787986456,
        "duration" : 0
      }
    }, {
      "uid" : "7119906dc5359e7b",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685787108235,
        "stop" : 1685787108235,
        "duration" : 0
      }
    }, {
      "uid" : "dda86dc32fee0055",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685786853231,
        "stop" : 1685786853231,
        "duration" : 0
      }
    }, {
      "uid" : "47584d57376bef77",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685786779007,
        "stop" : 1685786779007,
        "duration" : 0
      }
    }, {
      "uid" : "5de35e276e2c80e0",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685786612806,
        "stop" : 1685786612806,
        "duration" : 0
      }
    }, {
      "uid" : "c02e5a84abf85365",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685786493058,
        "stop" : 1685786493058,
        "duration" : 0
      }
    }, {
      "uid" : "43925c1ac289d528",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685786342295,
        "stop" : 1685786342295,
        "duration" : 0
      }
    }, {
      "uid" : "df7e5ca459e523b4",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685786240717,
        "stop" : 1685786240717,
        "duration" : 0
      }
    }, {
      "uid" : "a3f2d6fbd1068fc5",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685786174615,
        "stop" : 1685786174615,
        "duration" : 0
      }
    }, {
      "uid" : "ad65d21bf143d2c4",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685786117719,
        "stop" : 1685786117719,
        "duration" : 0
      }
    }, {
      "uid" : "70e6b23ec505e279",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685785818010,
        "stop" : 1685785818010,
        "duration" : 0
      }
    }, {
      "uid" : "62670e577fff4add",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685785700069,
        "stop" : 1685785700069,
        "duration" : 0
      }
    }, {
      "uid" : "ca26aacc8da52193",
      "status" : "skipped",
      "statusDetails" : "Skipped: unconditional skip",
      "time" : {
        "start" : 1685785614697,
        "stop" : 1685785614697,
        "duration" : 0
      }
    }, {
      "uid" : "6e3a960f58119f5",
      "status" : "passed",
      "time" : {
        "start" : 1685288163522,
        "stop" : 1685288167659,
        "duration" : 4137
      }
    }, {
      "uid" : "7b4a0882fb1ff69f",
      "status" : "broken",
      "statusDetails" : "TypeError: assert_equal() missing 1 required positional argument: 'self'",
      "time" : {
        "start" : 1685288025235,
        "stop" : 1685288029389,
        "duration" : 4154
      }
    }, {
      "uid" : "4ce926f552433ea1",
      "status" : "broken",
      "statusDetails" : "TypeError: assert_equal() missing 1 required positional argument: 'expected'",
      "time" : {
        "start" : 1685287824539,
        "stop" : 1685287828687,
        "duration" : 4148
      }
    }, {
      "uid" : "2cfa861d88ad2e30",
      "status" : "broken",
      "statusDetails" : "TypeError: assert_equal() missing 1 required positional argument: 'expected'",
      "time" : {
        "start" : 1685287689724,
        "stop" : 1685287693880,
        "duration" : 4156
      }
    }, {
      "uid" : "2e1d12b11a696272",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: 'index' attribute is unknown for the element. Only the following attributes are supported: [checkable, checked, {class,className}, clickable, {content-desc,contentDescription}, enabled, focusable, focused, {long-clickable,longClickable}, package, password, {resource-id,resourceId}, scrollable, selection-start, selection-end, selected, {text,name}, bounds, displayed, contentSize]\nStacktrace:\nio.appium.uiautomator2.common.exceptions.NoSuchAttributeException: 'index' attribute is unknown for the element. Only the following attributes are supported: [checkable, checked, {class,className}, clickable, {content-desc,contentDescription}, enabled, focusable, focused, {long-clickable,longClickable}, package, password, {resource-id,resourceId}, scrollable, selection-start, selection-end, selected, {text,name}, bounds, displayed, contentSize]\n\tat io.appium.uiautomator2.utils.ElementHelpers.generateNoAttributeException(ElementHelpers.java:167)\n\tat io.appium.uiautomator2.model.UiObject2Element.getAttribute(UiObject2Element.java:141)\n\tat io.appium.uiautomator2.handler.GetElementAttribute.safeHandle(GetElementAttribute.java:24)\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:41)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:261)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:255)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)\n\tat io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)\n\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)\n\tat io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)\n\tat io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:514)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:468)\n\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)\n\tat io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)\n\tat io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)\n\tat java.lang.Thread.run(Thread.java:818)",
      "time" : {
        "start" : 1685287363126,
        "stop" : 1685287367271,
        "duration" : 4145
      }
    }, {
      "uid" : "fe44cba6ec2bf0c4",
      "status" : "broken",
      "statusDetails" : "ValueError: invalid literal for int() with base 10: '支出'",
      "time" : {
        "start" : 1685287036865,
        "stop" : 1685287041764,
        "duration" : 4899
      }
    }, {
      "uid" : "9bfea09c1d2156aa",
      "status" : "broken",
      "statusDetails" : "ValueError: invalid literal for int() with base 10: '支出'",
      "time" : {
        "start" : 1685286948694,
        "stop" : 1685286952295,
        "duration" : 3601
      }
    }, {
      "uid" : "5f1ac83d73e39ff2",
      "status" : "broken",
      "statusDetails" : "ValueError: invalid literal for int() with base 10: '支出'",
      "time" : {
        "start" : 1685286777672,
        "stop" : 1685286782706,
        "duration" : 5034
      }
    }, {
      "uid" : "c97ff73a10884a18",
      "status" : "broken",
      "statusDetails" : "ValueError: invalid literal for int() with base 10: '支出'",
      "time" : {
        "start" : 1685286488116,
        "stop" : 1685286495572,
        "duration" : 7456
      }
    }, {
      "uid" : "e2482a7c26c4f397",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685286348324,
        "stop" : 1685286348324,
        "duration" : 0
      }
    }, {
      "uid" : "6de7547be2942aa7",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685286200439,
        "stop" : 1685286200439,
        "duration" : 0
      }
    }, {
      "uid" : "5b93d11dfdb39eb8",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685286026912,
        "stop" : 1685286026912,
        "duration" : 0
      }
    }, {
      "uid" : "e88761c6e6a63bfe",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685285829645,
        "stop" : 1685285829645,
        "duration" : 0
      }
    }, {
      "uid" : "32972a4ee5501bde",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0D8D2530>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1685285740516,
        "stop" : 1685285740516,
        "duration" : 0
      }
    }, {
      "uid" : "c2944643dbf513cf",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685285409164,
        "stop" : 1685285409164,
        "duration" : 0
      }
    }, {
      "uid" : "ddc06e648c777f2",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685285323884,
        "stop" : 1685285323884,
        "duration" : 0
      }
    }, {
      "uid" : "6784c2f080966b9b",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685284399128,
        "stop" : 1685284399128,
        "duration" : 0
      }
    }, {
      "uid" : "26026b5b756d20b1",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685284346134,
        "stop" : 1685284346134,
        "duration" : 0
      }
    }, {
      "uid" : "9e0fec37665bdbd8",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685284240991,
        "stop" : 1685284240991,
        "duration" : 0
      }
    }, {
      "uid" : "625276b76cf6955e",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685284032661,
        "stop" : 1685284032661,
        "duration" : 0
      }
    }, {
      "uid" : "3c3b0623ee6bf1d7",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685283949089,
        "stop" : 1685283949089,
        "duration" : 0
      }
    }, {
      "uid" : "8ef3fbaa8de8f897",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685283737765,
        "stop" : 1685283737765,
        "duration" : 0
      }
    }, {
      "uid" : "91bbb95648c989d6",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685283668380,
        "stop" : 1685283668380,
        "duration" : 0
      }
    }, {
      "uid" : "6074abd46a47aa93",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685283587054,
        "stop" : 1685283587054,
        "duration" : 0
      }
    }, {
      "uid" : "908cc50e02ec5e04",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0DC515B0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1685283465769,
        "stop" : 1685283465769,
        "duration" : 0
      }
    }, {
      "uid" : "2511fb00ba345ce7",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685283366350,
        "stop" : 1685283366350,
        "duration" : 0
      }
    }, {
      "uid" : "4f011526fb9e2306",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685283168546,
        "stop" : 1685283168546,
        "duration" : 0
      }
    }, {
      "uid" : "4badd573cfc401aa",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685283031054,
        "stop" : 1685283031054,
        "duration" : 0
      }
    }, {
      "uid" : "fc1458d0cdd0b8e0",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685282837381,
        "stop" : 1685282837381,
        "duration" : 0
      }
    }, {
      "uid" : "23a640881b1da6c1",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: pkg, and activity or intent action, are required to start an application\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: pkg, and activity or intent action, are required to start an application\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685282619539,
        "stop" : 1685282619539,
        "duration" : 0
      }
    }, {
      "uid" : "d15ec074d198dfac",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685282533082,
        "stop" : 1685282533082,
        "duration" : 0
      }
    }, {
      "uid" : "fcf65d5535af364f",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators/.CalculatorActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators/.CalculatorActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685282368379,
        "stop" : 1685282368379,
        "duration" : 0
      }
    }, {
      "uid" : "8bf749c3ac2d9daf",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start '.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685282285843,
        "stop" : 1685282285843,
        "duration" : 0
      }
    }, {
      "uid" : "947910ee60ffb94b",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators.com.ibox.calculators.SplashActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators.com.ibox.calculators.SplashActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685282094833,
        "stop" : 1685282094833,
        "duration" : 0
      }
    }, {
      "uid" : "9583af6f7b3f3012",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685282009783,
        "stop" : 1685282009783,
        "duration" : 0
      }
    }, {
      "uid" : "7302e05804b1bc41",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: The permission to start 'com.ibox.calculators.CalculatorActivity' activity has been denied.Make sure the activity/package names are correct.\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685281910754,
        "stop" : 1685281910754,
        "duration" : 0
      }
    }, {
      "uid" : "a3a029af7cc2bbee",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685281864433,
        "stop" : 1685281864433,
        "duration" : 0
      }
    }, {
      "uid" : "190a736734c3ad3f",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685281826509,
        "stop" : 1685281826509,
        "duration" : 0
      }
    }, {
      "uid" : "f718da129e058992",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0D31A630>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1685281485052,
        "stop" : 1685281485052,
        "duration" : 0
      }
    }, {
      "uid" : "15ad191f7c8f5742",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685268519535,
        "stop" : 1685268519535,
        "duration" : 0
      }
    }, {
      "uid" : "debeb70484ae8ae6",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators/com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators/com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685268390561,
        "stop" : 1685268390561,
        "duration" : 0
      }
    }, {
      "uid" : "37705bb88c59f0ad",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators/com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity name '.com.ibox.calculators/com.ibox.calculators.CalculatorActivity}' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685268321133,
        "stop" : 1685268321133,
        "duration" : 0
      }
    }, {
      "uid" : "52beb05e552cda64",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685268064585,
        "stop" : 1685268064585,
        "duration" : 0
      }
    }, {
      "uid" : "82314e186516a363",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685267963226,
        "stop" : 1685267963226,
        "duration" : 0
      }
    }, {
      "uid" : "b03b4f9432c7a611",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685267661528,
        "stop" : 1685267661528,
        "duration" : 0
      }
    }, {
      "uid" : "27583233ec55abb7",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685267435465,
        "stop" : 1685267435465,
        "duration" : 0
      }
    }, {
      "uid" : "197015cacd76671d",
      "status" : "broken",
      "statusDetails" : "AttributeError: 'NoneType' object has no attribute 'click'",
      "time" : {
        "start" : 1685267323734,
        "stop" : 1685267354362,
        "duration" : 30628
      }
    }, {
      "uid" : "c36a189832307c8a",
      "status" : "broken",
      "statusDetails" : "ValueError: invalid literal for int() with base 10: '支出'",
      "time" : {
        "start" : 1685267178132,
        "stop" : 1685267180555,
        "duration" : 2423
      }
    }, {
      "uid" : "4c65d896f0d96af8",
      "status" : "broken",
      "statusDetails" : "ValueError: invalid literal for int() with base 10: '支出'",
      "time" : {
        "start" : 1685267104865,
        "stop" : 1685267106855,
        "duration" : 1990
      }
    }, {
      "uid" : "8a21e0f31426ef0",
      "status" : "broken",
      "statusDetails" : "AttributeError: 'NoneType' object has no attribute 'click'",
      "time" : {
        "start" : 1685266829984,
        "stop" : 1685266829992,
        "duration" : 8
      }
    }, {
      "uid" : "cc86baf6609d7afc",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685264785417,
        "stop" : 1685264785417,
        "duration" : 0
      }
    }, {
      "uid" : "4df916eb955e0dca",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685264657335,
        "stop" : 1685264657335,
        "duration" : 0
      }
    }, {
      "uid" : "46007dba790667a7",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: Activity for intent 'android.intent.action.MAIN' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685264557755,
        "stop" : 1685264557755,
        "duration" : 0
      }
    }, {
      "uid" : "fabc8c7f60a0989",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685264364484,
        "stop" : 1685264364484,
        "duration" : 0
      }
    }, {
      "uid" : "f37724d0e8c32798",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.ibox.calculators' application. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting. Original error: 'com.ibox.calculators.SplashActivity' or 'com.ibox.calculators.com.ibox.calculators.SplashActivity' never started. Visit https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md for troubleshooting\n    at getResponseForW3CError (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\errors.js:804:9)\n    at asyncHandler (C:\\Program Files\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\protocol\\protocol.js:380:37)",
      "time" : {
        "start" : 1685264210319,
        "stop" : 1685264210319,
        "duration" : 0
      }
    } ],
    "categories" : [ ],
    "tags" : [ "@pytest.mark.skip" ]
  },
  "source" : "2572e3883aa7d889.json",
  "parameterValues" : [ "{'name': 'case002_参数化加法', 'num1': 70, 'num2': 32}" ]
}